config/secret_config.go

```diff
@@ -0,0 +1,99 @@
+package config
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/jahvon/tuikit/types"
+	"gopkg.in/yaml.v3"
+)
+
+type SecretConfig struct {
+	Name   string `json:"name"  yaml:"name"`
```

Comment:
```suggestion
	Reference   string `json:"reference"  yaml:"reference"`
```

Something like this would be more consistent with other naming of thus value

Reply:
Yes, I was only guessing what the fields names could be. Thanks